documentation/metadata-editing.md

```diff
@@ -0,0 +1,87 @@
+# Editing metadata of resources
+
+## What is a metadata resource
+
+Metadata resources contain auxiliary information about a resource. This indicates that a resource will always have a corresponding metadata resource and therefore metadata resources can not be created or deleted directly.
+
+Metadata resources are discoverable through interacting with its corresponding resource.
+More concrete, when performing a `GET` or `HEAD` request on a resource, there will be a `describedby` response **Link Header** with an URL that points to the metadata resource.
+
+**Note**: with the default configuration, a metadata resource that corresponds to a resource with URL `http://example.org/resource` will have `http://example.org/resource.meta` as URL. 
+However, this does not have to be always the case as the Solid Protocol does not state anything about the strategy of encoding the metadata resource. Therefore, the CSS makes this configurable in the [metadata configuration file](../config/util/auxiliary/strategies/meta.json).
+
+
+
+## How to edit the metadata of a resource
+
+Editing the metadata of a resource is performed by editing the auxiliary metadata resource directly. This can only be done using `PATCH` requests (see [example workflow](#workflow)).
+Thus it is forbidden to perform a `PUT` request on metadata. The reason of this approach is that some metadata is [protected by the server](https://solidproject.org/TR/protocol#resource-containment) and other metadata is generated by the server. A `PUT` request, according to the HTTP specification, replaces the state of the target resource entirely by the request payload. This is not possible with metadata resources and therefore not allowed.
+
+### Protected metadata
+
+There is some metadata that is forbidden to be modified. The CSS will throw an error when trying to change this protected metadata.
+
+Therefore any change PATCH request with a query that will result into modifying the **storage indication** or the **containment triples** will result into a 409 `ConflictHttpError`.
+
+#### Storage indication
```

Comment:
We shouldn't really list everything that the user is not allowed to do. Just keep it to how they can do what they want.